modules/displayTodos.js

```diff
@@ -1,31 +1,30 @@
-import '../src/style.css';
-
-const DisplayTodos = () => {
-  let todos = JSON.parse(localStorage.getItem('todos')) || [];
+import { editItem, deleteItem } from './addEditDelete.js';
 
+const displayTodos = () => {
+  const todos = JSON.parse(localStorage.getItem('todos')) || [];
```

Comment:
- [ ] You should test all the functions responsible for additem and deleteItem so you should also test this function and use `Mocks` for `local storage` as in the requirements 
![image](https://user-images.githubusercontent.com/61790587/172919778-4b243cc5-ee0f-4909-b826-75648b7ee94c.png)